components/button/index.css

```diff
@@ -327,13 +346,10 @@ a.spectrum-Button {
 			--highcontrast-button-background-color-hover: Highlight;
 			--highcontrast-button-background-color-down: Highlight;
 			--highcontrast-button-background-color-focus: Highlight;
-			--highcontrast-button-content-color-hover: ButtonFace;
-			--highcontrast-button-content-color-down: ButtonFace;
-			--highcontrast-button-content-color-focus: ButtonFace;
-
-			.spectrum-Button-label {
-				forced-color-adjust: none;
-			}
+			--highcontrast-button-border-color-default: ButtonText;
+			--highcontrast-button-border-color-hover: Highlight;
+			--highcontrast-button-border-color-focus: Highlight;
+			--highcontrast-button-border-color-down: Highlight;
```

Comment:
It looks like the latest update lost the 3 properties for content-color in `&.spectrum-Button--accent.spectrum-Button--fill` (set to `HighlightText`) that you had before, so I am seeing the wrong foreground on accent for hovered, focused, and active.